src/pages/invite.js

```diff
@@ -84,7 +106,8 @@ class InvitePage extends React.Component {
             title={this.state.title}
             description={this.state.description}
             amount={this.state.amount}
-            time={this.state.time} />
+            time={this.state.time}
+            imageUrl={this.imageUrl()} />
```

Comment:
Actually it seems we need to call the image URL to get the image first, otherwise, the transformed image is not generated so the image is shown as blank.